build/yamls/flow-collector/logstash/ipfix.yml

```diff
@@ -0,0 +1,4056 @@
+---
```

Comment:
Again how this file is generated?

Reply:
It is modified based on [ipfix file](https://github.com/logstash-plugins/logstash-codec-netflow/blob/master/lib/logstash/codecs/netflow/ipfix.yaml) from logstash Netflow codec plugin to interpret the fields from flow records. I added Antrea-specific fields at the end of the file and modified 2 data types.